providers.md

```diff
@@ -12,7 +12,7 @@
 
 Service providers are the central place of all Laravel application bootstrapping. Your own application, as well as all of Laravel's core services are bootstrapped via service providers.
 
-But, what do we mean by "bootstrapped"? In general, we mean **registering** things, including registering service container bindings, event listeners, middleware, and even routes. Service providers are the central place to configure your application.
+But, what do we mean by "bootstrapped"? In general, we mean **registering** things, including registering service container bindings, event listeners, middlewares, and even routes. Service providers are the central place to configure your application.
```

Comment:
This isn't correct. Middleware is already the plural.

Reply:
@driesvints thanks for the review. I rollbacked my this change.